### DISCUSSION TITLE
Is there a way to show a different glyph depending on the type of project for the project segment?

### DISCUSSION THREAD

**Discussion by paulomorgado** [JanDeDobbeleer/oh-my-posh#4427] [General]
I want to show different glyphs depending on the type of project for the project segment.

Is that possible?

**Comment by JanDeDobbeleer** [comment]
You can make use of the `.Type` property in the template to adjust the icon.

**Reply by paulomorgado** [reply]
How can I do that? `{{ if }}` seems to test only if it's empty or not.

**Reply by JanDeDobbeleer** [reply]
@paulomorgado you can check for values as well using `{{ if eq .Type \"dotnet\" }}` for example. Templates also have else functionality. See [here](https://ohmyposh.dev/docs/configuration/templates#template-logic) for a few examples.

**Reply by paulomorgado** [reply]
I ended up trying `{{ if regexMatch \"^dotnet$\" .Type }}\ue70c{{ else }}\ue70d{{ end }}`, but it's overkill to use regular expressions for this.

Now that I know how to do it, I found it here: https://pkg.go.dev/text/template#hdr-Text_and_spaces

**Comment by paulomorgado** [comment]
Check it out here: #4428
